build.gradle

```diff
@@ -33,6 +35,17 @@ googleJavaFormat {
   exclude 'build/**'
 }
 
+if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
+    project.name != 'benchmark') {
```

Comment:
We need to make sure that someone can clone gax-java & call `./gradlew build` without having `ossrhUsername`/`ossrhPassword` set.